### PR TITLE
Update CCC_TAC_Minutes-2022-04-07.md

### PR DESCRIPTION
**Pull request by krtaylor**
Minutes from 4-7 TAC meeting

### CODE REVIEW

File: TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-# Confidential Computing Consortium TAC meeting
+# Confidential Computing Consortium 
+##Minutes of the Technical Advisory Council
```
Comment (krtaylor) [inline TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md:2]: Formatting for sub-heading broken, I will fix in next revision

File: TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md [hunk 2]
```diff
@@ -41,89 +42,117 @@
...
+* Kurt Taylor (Linux Foundation)
+* Helen Lau (Linux Foundation)        
 
 ### Meeting minute approvals
 
 #### Approved by voting members
 
-<!--
-**RESOLVED:** That the minutes of the [March 10, 2022](../2022-03-10/CCC_TAC_Minutes-2022-03-10.md) meeting of the Technical Advisory Council meeting of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.-->
+**Resolved:** That the minutes of the [March 10, 2022](../2022-03-10/TAC_Minutes-2022-03-10.pdf) meeting of the Technical Advisory Council meeting of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
```
Comment (dcmiddle) [inline TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md:83]: s/pdf/md
https://github.com/confidential-computing/governance/blob/main/TAC/Meetings/2022/2022-03-10/CCC_TAC_Minutes-2022-03-10.md

File: TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md [hunk 3]
```diff
@@ -41,89 +42,117 @@
...
-1. **[Mike, Eric V, Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
+1. **[Mike, Eric, Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
 
 1. **[Stephen Walli, Dan Middleton, Aeva Black]** Develop a CoC escalation process, to be presented to TAC and governing board.
+    * Dan Middleton has opened a PR for review
 
-1. **[Brian Warner]** Investigate prior efforts to finalize Graduation lifecycle language, and if needed file an issue to finalize Graduation lifecycle language.
-
-1. **[Dave Thaler]** Open an issue requesting someone sync project benefits slides to the repo.
+1. **[Mike, Eric V, Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
```
Comment (dcmiddle) [inline TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md:103]: duplicate of line 98

File: TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md [hunk 4]
```diff
@@ -41,89 +42,117 @@
...
-### Open Action Items
 
-1. **[Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
+### Open Action Items
 
-1. **[Mike, Eric V, Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
+1. **[Mike, Eric, Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
 
 1. **[Stephen Walli, Dan Middleton, Aeva Black]** Develop a CoC escalation process, to be presented to TAC and governing board.
+    * Dan Middleton has opened a PR for review
```
Comment (dcmiddle) [inline TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md:101]:
```suggestion
    * Dan Middleton has opened a PR for review https://github.com/confidential-computing/governance/pull/92
```

File: TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md [hunk 5]
```diff
@@ -41,89 +42,117 @@
...
+### Completed Action Items
+1. **[Dan Middleton]** Work with CCC Attestation to schedule annual review, tentatively for March 24, 2022.
 
-* Veraison
+1. **[Dave Thaler]** Open an issue requesting someone sync project benefits slides to the repo.
 
+1. **[Brian Warner]** Update the Google Sheet containing the [list of CCC dependencies](https://docs.google.com/spreadsheets/d/1UKnbbGWXYLjnPZsox3zmYo59nv3XSXjePfas5E2fER0/edit#gid=0) with the changed dependencies noted in the project review document.
 
-### TAC Tech Talk - Confidential Computing from the perspective of the network
+1. **[Brian Warner]** Add Enarx annual review to the agenda for March 10th TAC meeting.
```
Comment (dcmiddle) [inline TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md:118]: Is this maybe a copy paste from an earlier minutes?

File: TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md [hunk 6]
```diff
@@ -41,89 +42,117 @@
...
 
+1. **[Brian Warner]** Add slide to this deck reminding the TAC to review PRs and issues, time permitting.
 
-### CCC-Attestation annual review
+1. **[Brian Warner]** Schedule a deep-dive into LFX code scanning for April 19, 2022.
```
Comment (dcmiddle) [inline TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md:122]: s/19/21
```suggestion
1. **[Brian Warner]** Schedule a deep-dive into LFX code scanning for April 21, 2022.
```

**Review by dcmiddle — CHANGES_REQUESTED** [review]

File: TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md [hunk 7]
```diff
@@ -10,15 +11,15 @@
...
 
 ### Voting members of the TAC
 
 *Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*
 
 * **Accenture** - Giuseppe Giordano
-* **Ant Group** - Zongmin Gu / Tate, Hongliang Tian
-* **Arm** - Thomas Fossati / Michael Lu
+* **ANT Group** - Zongmin Gu
+* **ARM** - Thomas Fossati / Michael Lu
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md:22]: Revert.  The correct capitalization is "Arm".  See https://www.arm.com/company/policies/trademarks

And according to https://en.wikipedia.org/wiki/Ant_Group "Ant Group" was also correct as it was.

File: TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md [hunk 8]
```diff
@@ -41,89 +42,117 @@
...
 #### Voting members
 
-<!-- * Dan Middleton (Intel)
+* Simon Johnson (Intel)
 * Dave Thaler (Microsoft)
-* Eric Northup (Meta)
 * Lily Sturman (Red Hat)
-* Zhipeng (Howard) Huang (Huawei)
-* Thomas Fossati (Arm) -->
+* Thomas Fossati (ARM)
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md:54]:
```suggestion
* Thomas Fossati (Arm)
```

(Line 69 on the other hand is correct)

File: TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md (hunk 2 above)
Comment (dthaler) [reply TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md:83]:
```suggestion
**Resolved:** That the minutes of the [March 10, 2022](../2022-03-10/TAC_Minutes-2022-03-10.md) meeting of the Technical Advisory Council meeting of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
```

File: TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md [hunk 9]
```diff
@@ -41,89 +42,117 @@
...
+ * Quorum reached, Minutes of March 10, 2022 were approved
 
 #### Pending approval
+ * None
 
-* [March 10, 2022](../2022-03-10/CCC_TAC_Minutes-2022-03-10.md)
+### TAC Tech Talks
+ *  Mark Novak - CC Governance Framework
+ 	*  Mark N, Thomas F, Naveen C, Steve VL, and Eric N expressed interest in collaborating on the governance topic, SIG formation possible
+ *  Eric Voit, Penglin Yang - Networking Confidential Computing
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md:93]: Please update minutes with links to the slide decks

File: TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md [hunk 10]
```diff
@@ -41,89 +42,117 @@
...
 
-### Open Action Items
 
-1. **[Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
+### Open Action Items
 
-1. **[Mike, Eric V, Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
+1. **[Mike, Eric, Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md:98]: Revert.  We have two Eric's (Eric N and Eric V) that are regular attendees so need to make sure the minutes accurately reflect which one in minutes.  (Line 103 still has the V so is fine)

File: TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md [hunk 11]
```diff
@@ -41,89 +42,117 @@
...
-1. **[Brian Warner]** Investigate prior efforts to finalize Graduation lifecycle language, and if needed file an issue to finalize Graduation lifecycle language.
-
-1. **[Dave Thaler]** Open an issue requesting someone sync project benefits slides to the repo.
+1. **[Mike, Eric V, Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
+    * Open Enclave maintainers are close to 100% trained (Dave Thaler)
+    * Veracruz lead has taken the course (Thomas Fossati).
+    * Please report back that project maintainers have been made aware of the training, and have been asked to take it.
 
-### Completed Action Items
+1. **[Kurt Taylor]** Investigate prior efforts to finalize Graduated lifecycle language, and if needed file an issue to finalize Graduated lifecycle language. (DT) summarized options for resolution: 1) Adopt new stages 2) talk about proposal and fix it 3) delete the proposed new terms and just use the the two currently adopted stages, sandbox and incubation
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md:108]:
```suggestion
1. **[Kurt Taylor]** Investigate prior efforts to finalize Graduated lifecycle language, and if needed file an issue to finalize Graduated lifecycle language. (DT) summarized options for resolution: 1) Adopt new stages 2) talk about proposal and fix it 3) delete the proposed new terms and just use the two currently adopted stages, sandbox and incubation
```

File: TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md [hunk 12]
```diff
@@ -41,89 +42,117 @@
...
 
-1. **[Brian Warner]** Verify that third-party dependencies appear in the [CCC dependencies spreadsheet](https://docs.google.com/spreadsheets/d/1UKnbbGWXYLjnPZsox3zmYo59nv3XSXjePfas5E2fER0/edit#gid=0). (done)
 
-### Project overviews
+### Completed Action Items
+1. **[Dan Middleton]** Work with CCC Attestation to schedule annual review, tentatively for March 24, 2022.
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md:112]:
```suggestion
1. **[Dan Middleton]** Work with CCC Attestation to schedule annual review, tentatively for April 21, 2022.
```

File: TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md [hunk 13]
```diff
@@ -41,89 +42,117 @@
...
 
-* Veraison
+1. **[Dave Thaler]** Open an issue requesting someone sync project benefits slides to the repo.
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md:114]:
```suggestion
1. **[Dave Thaler]** Open an issue requesting someone sync project benefits slides to the repo.  (Done: https://github.com/confidential-computing/governance/issues/94)
```

File: TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md [hunk 14]
```diff
@@ -41,89 +42,117 @@
...
+1. **[Brian Warner]** Schedule a deep-dive into LFX code scanning for April 19, 2022.
+    * Confirmed, Shubhra Kar and David Deal from LF to present.
 
+### Project overviews
 
-### Common terminology
 
+### Interns
+* Nick Vidal (NV) update on Interns, slack #general discussion, help and guidance 
+* Deferr to next meeting when Dan Middleton will be participating
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md:130]:
```suggestion
* Defer to next meeting when Dan Middleton will be participating 
```

File: TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md [hunk 15]
```diff
@@ -41,89 +42,117 @@
...
+* (DT) discussed needed updates to the Trusted Computing wikipedia page, also create a CCC or Confidential Computing page (none currently exists)
+	* (NV) suggested the interns do an initial wikipedia page for CC, (NV) will work with them 
+	* (NV) authoring a blog, will announce soon	 
 
-### Future topics
+### Common terminology
+ * (DT) discussed Google Doc for terminology
+ * Eric Voit (EV) raised multi node question as previously discussed with regard to confidential cloud, multi node is a hard problem to define, will defer to a future version
+ * Steve Van Lare (SVL) will respond to comments in document
+ * (DT) proposed and (SVL) discussed confidential workload/application - protects all legs of CC, (at rest, in motion, and in use)
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md:146]:
```suggestion
 * (DT) proposed and (SVL) discussed confidential workload/application - protects all legs of data protection (at rest, in motion, and in use) 
```

File: TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md [hunk 16]
```diff
@@ -41,89 +42,117 @@
...
+	* (NV) suggested the interns do an initial wikipedia page for CC, (NV) will work with them 
+	* (NV) authoring a blog, will announce soon	 
 
-### Future topics
+### Common terminology
+ * (DT) discussed Google Doc for terminology
+ * Eric Voit (EV) raised multi node question as previously discussed with regard to confidential cloud, multi node is a hard problem to define, will defer to a future version
+ * Steve Van Lare (SVL) will respond to comments in document
+ * (DT) proposed and (SVL) discussed confidential workload/application - protects all legs of CC, (at rest, in motion, and in use) 
+ * (EV) github process closure (DT) proposed PDF to whitepaper, Draft then to LF creative team for editing process, goal to be firmed up by next meeting, draft by second meeting will be signed off, I'll find out creative services timeline
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md:147]:
```suggestion
 * (EV) github process closure (DT) proposed PDF to whitepaper, Draft then to LF creative team for editing process, goal to be firmed up by next meeting, draft by second meeting will be signed off, KT to find out creative services timeline
```
Also this means KT has an action item that should be listed in the action item list

**Review by dthaler — CHANGES_REQUESTED** [review]

File: TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md (hunk 9 above)
Comment (krtaylor) [reply TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md:93]: The slides and agenda are in the same meeting folder as the minutes

Comment (dthaler) [reply TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md:93]: Thats why it should be trivial to link them.
```suggestion
 *  Mark Novak - [Confidential Computing Governance](Confidential%20Computing%20Governance.pdf)
 	*  Mark N, Thomas F, Naveen C, Steve VL, and Eric N expressed interest in collaborating on the governance topic, SIG formation possible
 *  Eric Voit, Penglin Yang - [Networking Confidential Computing](CCC-Networking-Tech-Talk-7Apr2022.pdf)
```

**Review by dthaler — APPROVED** [review]

**Review by dcmiddle — APPROVED** [review]

File: TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md (hunk 3 above)
Comment (dthaler) [reply TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md:101]: I believe the list of names in in red above is correct (Mike and Eric did their action items)

Comment (krtaylor) [inline TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md:101]:
```suggestion
1. **[Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
```

**Pull request merged**